src/main/kotlin/com/terraformation/backend/species/db/GbifStore.kt

```diff
@@ -100,7 +100,6 @@ class GbifStore(private val dslContext: DSLContext) {
         .orderBy(firstWordSortPosition, GBIF_NAMES.NAME)
         .limit(maxResults)
         .fetchInto(GbifNamesRow::class.java)
-        .filterNotNull()
```

Comment:
this removal intended?

Reply:
It was unnecessary. `fetchInto(T::class.java)` returns a list that never has null values. This call was probably left over from an earlier draft of the code.